.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com
+# REACT_APP_EGGHEAD_BASE_URL=http://localhost:5000
```

Comment:
You can uncomment the environment you want to use. Staging just works. Dev requires setup of `egghead-rails` locally.